core/src/main/java/com/linecorp/armeria/client/RefreshingAddressResolver.java

```diff
@@ -65,7 +66,7 @@
     private volatile boolean resolverClosed;
 
     RefreshingAddressResolver(EventLoop eventLoop,
-                              ConcurrentMap<String, CompletableFuture<CacheEntry>> cache,
+                              @Nullable Cache<String, CompletableFuture<CacheEntry>> cache,
```

Comment:
Question: Do we need `RefreshingAddressResolver` when `Cache` is null?

Reply:
I was concerned about delays in updating a DNS record if someone uses DNS failover.
But it seems too much to be considered here. So I modified it to work with Nonnull.